### ISSUE TITLE
0.1.10 postinstall script failing

### ISSUE THREAD

**Issue by dsifford** [theia-ide/typescript-language-server#39]
```
147 silly postinstall command-exists@1.2.6
148 info lifecycle command-exists@1.2.6~postinstall: command-exists@1.2.6
149 silly postinstall typescript-language-server@0.1.10
150 info lifecycle typescript-language-server@0.1.10~postinstall: typescript-language-server@0.1.10
151 verbose lifecycle typescript-language-server@0.1.10~postinstall: unsafe-perm in lifecycle true
152 verbose lifecycle typescript-language-server@0.1.10~postinstall: PATH: /usr/lib/node_modules/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/dsifford/.local/share/npm/lib/node_modules/typescript-language-server/node_modules/.bin:/home/dsifford/.local/share/npm/lib/node_modules/.bin:/home/dsifford/.local/share/npm/lib/node_modules/node_modules/.bin:/home/dsifford/.local/share/npm/lib/node_modules/.bin:/home/dsifford/.local/share/npm/node_modules/.bin:/home/dsifford/.local/share/node_modules/.bin:/home/dsifford/.local/node_modules/.bin:/home/dsifford/node_modules/.bin:/home/node_modules/.bin:/node_modules/.bin:/usr/bin:/home/dsifford/.local/bin:/home/dsifford/.local/share/npm/bin:/home/dsifford/gocode/bin:/usr/local/bin:/usr/local/sbin:/usr/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/bin:/home/dsifford/.dotfiles/vim/_plugins/fzf/bin
153 verbose lifecycle typescript-language-server@0.1.10~postinstall: CWD: /home/dsifford/.local/share/npm/lib/node_modules/typescript-language-server
154 silly lifecycle typescript-language-server@0.1.10~postinstall: Args: [ '-c', 'yarn run build' ]
155 silly lifecycle typescript-language-server@0.1.10~postinstall: Returned: code: 1  signal: null
156 info lifecycle typescript-language-server@0.1.10~postinstall: Failed to exec postinstall script
157 verbose unlock done using /home/dsifford/.cache/npm/_locks/staging-4fbf0f0616f3d71e.lock for /home/dsifford/.local/share/npm/lib/node_modules/.staging
158 verbose stack Error: typescript-language-server@0.1.10 postinstall: `yarn run build`
158 verbose stack Exit status 1
158 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:283:16)
158 verbose stack     at EventEmitter.emit (events.js:180:13)
158 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
158 verbose stack     at ChildProcess.emit (events.js:180:13)
158 verbose stack     at maybeClose (internal/child_process.js:936:16)
158 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:220:5)
159 verbose pkgid typescript-language-server@0.1.10
160 verbose cwd /home/dsifford/.local/share/npm/lib/node_modules
161 verbose Linux 4.16.3-1-ARCH
162 verbose argv "/usr/bin/node" "/usr/bin/npm" "install" "--global" "typescript-language-server@0.1.10" "--color=always"
163 verbose node v9.11.1
164 verbose npm  v5.8.0
165 error code ELIFECYCLE
166 error errno 1
167 error typescript-language-server@0.1.10 postinstall: `yarn run build`
167 error Exit status 1
168 error Failed at the typescript-language-server@0.1.10 postinstall script.
168 error This is probably not a problem with npm. There is likely additional logging output above.
169 verbose exit [ 1, true ]
```

Possibly related: The `postinstall` script has a `yarn` dependency. Is that fragile? Should this instead by an explicit command?

**Comment by svenefftinge** [comment]
We don't need it at all. I am replacing it with `prepare`.

**Comment by svenefftinge** [comment]
fixed in https://github.com/theia-ide/typescript-language-server/commit/6a89510c90dd5ee6fc94396d864fe325c1da272c
I also published 0.1.11

**Comment by dsifford** [comment]
Thanks so much @svenefftinge! Greatly appreciate your hard work :pray:
